release-4.1.1: add Changes in release-4.1.1, Required Actions, Bugs

cclhsu opened the pull request:
release-4.1.1: add Changes in release-4.1.1, Required Actions, Bugs Fixes in release notes

Signed-off-by: Clark Hsu <clark.hsu@suse.com>

jordimassaguerpla reviewed (changes requested):
Let me hold this because we have an issue with 4.1.0, and I would like to merge that first.

Commit:
release-4.1.1: add Changes in release-4.1.1, Required Actions, Bugs Fixes

release-4.1.1: add Changes in release-4.1.1, Required Actions, Bugs Fixes in release notes

Signed-off-by: Clark Hsu <clark.hsu@suse.com>

jordimassaguerpla committed:
Merge branch 'master' into release-4.1.1
